test-tests/shared/src/test/scala/zio/test/DefaultTestReporterSpec.scala

```diff
@@ -1,32 +1,29 @@
-package zio.test
-
-import zio._
 import zio.clock.Clock
-import zio.test.environment._
 import zio.test.Assertion._
 import zio.test.ReportingTestUtils._
-import zio.test.TestUtils.label
+import zio.test._
+import zio.test.environment.{TestClock, TestConsole, TestEnvironment, testEnvironmentManaged}
+import zio.{Cause, IO, Managed, ZIO}
 
 import scala.concurrent.Future
 
-object DefaultTestReporterSpec extends AsyncBaseSpec {
-
-  val run: List[Async[(Boolean, String)]] = List(
-    label(reportSuccess, "correctly reports a successful test"),
-    label(reportFailure, "correctly reports a failed test"),
-    label(reportError, "correctly reports an error in a test"),
-    label(reportSuite1, "correctly reports successful test suite"),
-    label(reportSuite2, "correctly reports failed test suite"),
-    label(reportSuites, "correctly reports multiple test suites"),
-    label(simpleAssertion, "correctly reports failure of simple assertion"),
-    label(multipleNestedFailures, "correctly reports multiple nested failures")
-  )
+object DefaultTestReporterSpec
+    extends ZIOBaseSpec(
+      suite("DefaultTestReporterSpec")(
+        testM("report success") {
+          assert(ZIO.effect(DefaultTestReporterSpecUtil
```

Comment:
Since the new `runLog` method will be returning a `ZIO`, we can get rid of the `ZIO.effect` call. I would import the `DefaultTestReporterSpecUtil` object so you don't need to type it out every time. Then since `runLog` will be returning a `Vector[String]` you can assert that the result is the same as the expected result.